src/test/kotlin/com/hibob/academy/employeeFeedback/dao/FeedbackDaoTest.kt

```diff
@@ -178,8 +178,8 @@ class FeedbackDaoTest @Autowired constructor(private val sql: DSLContext) {
         )
 
         val feedbacks1 = feedbackDao.getFeedbacksByFilters(companyId, filter)
-        val feedbackAfterConvert = feedbacks1.map { it.toCreateFeedbackRequest() }
-        val exceptedFeedbacks = listOf(anonymousFeedback1, feedback1)
+        val feedbackAfterConvert = feedbacks1.map { it.toCreateFeedbackRequest() }.toSet()
```

Comment:
You can't assume ordering in DB, unless you explicitly order the response by something. 
toSet is one option, but you can use `must contain all elements of` (or something like that) instead.